CITATION.cff

```diff
@@ -0,0 +1,31 @@
+cff-version: 1.2.0
+message: "Please cite the following works when using this software."
```

Comment:
Is this software?

Reply:
Yes, but your point stands as it isn't general use software. Could do

```
message: "Please cite the following works when using this repository and talk."
```